### ISSUE TITLE
Cannot build for STM32L552ZE-Q

### ISSUE THREAD

**Issue by asukiaaa** [stm32duino/STM32FreeRTOS#51]
I tried building for STM32L552ZE-Q but failed with the following message.
```
/tmp/ccjgv9nE.s: Assembler messages:
/tmp/ccjgv9nE.s:224: Error: selected processor does not support `vstmdbeq r0!,{s16-s31}' in Thumb mode
/tmp/ccjgv9nE.s:225: Error: instruction not allowed in IT block -- `mrs r2,psplim'
/tmp/ccjgv9nE.s:250: Error: selected processor does not support `vldmiaeq r0!,{s16-s31}' in Thumb mode
/tmp/ccjgv9nE.s:252: Error: instruction not allowed in IT block -- `msr psplim,r2'
*** [.pio/build/nucleo_l552ze_q/lib67e/STM32duino FreeRTOS/portasm.c.o] Error 1
```

It seems similar to https://github.com/stm32duino/STM32FreeRTOS/issues/44 but STM32L5 is supported bu https://github.com/stm32duino/STM32FreeRTOS/commit/adba67836e25becd8ded5bf5a7ba09c2d616769c so I report this.

My environment
```
platformio 5.2.5
platform-ststm32 15.3.0
```

platformio.ini
```
[env:nucleo_l552ze_q]
platform = ststm32
board = nucleo_l552ze_q
framework = arduino
monitor_speed = 115200
lib_deps =
  STM32duino FreeRTOS
```

Is there any idea to avoid this error?

Thank you for sharing the useful library.

**Comment by fpistm** [comment]
Hi @asukiaaa ,

L5 support has been added and is available with STM32FreeRTOS 10.3.1.
With Arduino there is no issue. I don't know how PIO handle this but you probably don't have the latest release.
You should request support there as we support only Arduino IDE.

**Comment by asukiaaa** [comment]
I succeeded in building by adding `-mfpu=fpv4-sp-d16 -mfloat-abi=hard` for build commands so I created a PR.
https://github.com/stm32duino/Arduino_Core_STM32/pull/1704
